sigs.yaml

```diff
@@ -509,23 +509,30 @@ sigs:
       establishment of conventions for writing CLI commands, POSIX compliance,
       and improving the command line tools from a developer and devops user
       experience and usability perspective.
-    charter_link:
+    charter_link: https://github.com/kubernetes/community/blob/master/sig-cli/charter.md
```

Comment:
this can just be `charter_link: charter.md` :)

Also, can you squash into one commit? Thank you!